README.md

```diff
@@ -11,16 +11,16 @@ and transposed convolutions in the decoder.
 
 [Conv-Tasnet](https://arxiv.org/abs/1809.07454)
 is a separation model developed for speech which predicts a mask on a learnt over-complete linear representation
-using a purely convolutional model with stride of 1 and dilated convolutional blocks.
+using a purely convolutional model with the stride of 1 and dilated convolutional blocks.
 We reused the code from the [kaituoxu/Conv-TasNet][tasnet]
 repository and added support for multiple audio channels.
 
 
-When trained only on MusDB, Conv-Tasnet achieves higher SDR than Demucs (5.7 vs 5.6).
+When trained only on MusDB, Conv-Tasnet achieves higher SDR than Demucs (5.7 vs. 5.6).
 However, the audio it generates has significant artifacts as measured by human evaluations
 (MOS is 3.2 for Demucs, 2.9 for Conv-Tasnet). When trained with extra training data,
 Demucs and Conv-Tasnet obtain the same SDR. See [our paper][demucs_arxiv] Section 6 for more details or listen to our
-[audio samples][audio] .
+[audio samples][audio].
```

Comment:
This will make the `.` disappear because it is just next to a link. The space is not displayed by Markdown.

Reply:
No it renders fine in markdown.